pkg/kv/kvserver/stateloader/stateloader.go

```diff
@@ -337,6 +337,21 @@ func (rsl StateLoader) SetVersion(
 		hlc.Timestamp{}, hlc.ClockTimestamp{}, nil, version)
 }
 
+// UninitializedReplicaState returns the ReplicaState of an uninitialized
+// Replica with the given range ID. It is equivalent to StateLoader.Load from an
+// empty storage.
+func UninitializedReplicaState(rangeID roachpb.RangeID) kvserverpb.ReplicaState {
+	// TODO(pavelkalinnikov): figure out what to do with the Version field.
```

Comment:
Is there anything to figure out? A nil version is fine - the versioning is a property of the state machine, which is in zero state here.

Reply:
I mean, what if there is an uninit replica with `Version != nil`? Or are you saying that `Version` can be non-nil only for inited replicas?

Reply:
It doesn't look like we're using it at all at the moment (probably because there are no ongoing migrations below Raft).

Reply:
Removed the TODO.